ArcGISRuntimeSDKQt_CppSamples/Geometry/CutGeometry/CutGeometry.cpp

```diff
@@ -73,7 +73,8 @@ void CutGeometry::componentComplete()
 void CutGeometry::cutPolygon()
 {
   // perform the cut
-  QList<Geometry> geoms = GeometryEngine::cut(m_lakeSuperiorGraphic->geometry(), m_borderGraphic->geometry());
+  QList<Geometry> geoms = GeometryEngine::cut(m_lakeSuperiorGraphic->geometry(),
+                                              geometry_cast<Polyline>(m_borderGraphic->geometry()));
```

Comment:
Why are we only having to cast on the second graphics geometry?

Reply:
The signature of the method is:
```c++
GeometryEngine::cut(const Geometry& geometry, const Polyline& cutter);
```